extensions/amp-tiktok/validator-amp-tiktok.protoascii

```diff
@@ -0,0 +1,112 @@
+#
+# Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+
+tags: {  # amp-tiktok
+  html_format: AMP
+  tag_name: "SCRIPT"
+  extension_spec: {
+    name: "amp-tiktok"
+    version: "0.1"
+    version: "latest"
+  }
+  attr_lists: "common-extension-attrs"
+}
+tags: {  # <amp-tiktok>
+  html_format: AMP
+  tag_name: "AMP-TIKTOK"
+  spec_name:"AMP-TIKTOK video id"
+  requires_extension: "amp-tiktok"
+  attr_lists: "extended-amp-global"
+  attrs: {
+    name: 'data-src';
+    value_regex: '[0-9]+';
+  }
+  spec_url: "https://amp.dev/documentation/components/amp-tiktok"
+  amp_layout: {
+    supported_layouts: FILL
+    supported_layouts: FIXED
+    supported_layouts: FIXED_HEIGHT
+    supported_layouts: FLEX_ITEM
+    supported_layouts: INTRINSIC
+    supported_layouts: NODISPLAY
+    supported_layouts: RESPONSIVE
+  }
+}
+tags: {  # <amp-tiktok>
+  html_format: AMP
+  tag_name: "AMP-TIKTOK"
+  spec_name:"AMP-TIKTOK video source url"
+  requires_extension: "amp-tiktok"
+  attr_lists: "extended-amp-global"
+  attrs: {
+    name: 'data-src';
```

Comment:
It seems a little nicer to make different attribute names for the id vs the source url: that way the runtime doesn't have to rely on heuristics to figure out which one is being used. I don't feel strongly here; just my spidey sense to avoid heuristics in general, because edge cases. But this seems unlikely in this case so :shrug:

Reply:
@alanorozco Do you think this would be worth updating before we release the component?

Reply:
Resolving; looks like this was addressed, with the URL form being moved to the inner `<blockquote cite>`. LMK if I misunderstood.

Reply:
(Oh I think I misunderstood. id is required now. Either way, resolved.)

Reply:
Resolved; looks like this was addressed by moving the URL into `<blockquote cite>` with the separate `amp-tiktok` spec for each case.